src/ralph/admin/filters.py

```diff
@@ -40,6 +66,9 @@ def has_output(self):
     def lookups(self, request, model_admin):
         return None
 
+    def expected_parameters(self):
```

Comment:
ok, never mind, i've found `Returns the list of parameter names that are expected from the request’s query string and that will be used by this filter.`